client/src/App.tsx

```diff
@@ -23,6 +24,9 @@ function App(): JSX.Element {
                 <Route exact path="/dashboard">
                   <Dashboard />
                 </Route>
+                <Route exact path="/manage_bookings">
```

Comment:
```suggestion
                <Route exact path="/manage-bookings">
```
By convention, we use hyphenated path names 👍🏼